trl/trainer/ppov2_trainer.py

```diff
@@ -599,7 +604,7 @@ def repeat_generator():
 
     def generate_completions(self, sampling: bool = False):
         args = self.args
-        tokenizer = self.tokenizer
+        processing_class = self.processing_class
```

Comment:
I may be missing something but is this required? Cannot we just use `self.processing_class`?

Reply:
You're right. Same for `args`. It will probably need some refactoring in the future

Reply:
I've seen that in other places too so maybe there's a rationale that I don't see for that? Not sure, but sure we'll keep it in mind